lib/github.sh

```diff
@@ -207,6 +206,23 @@ strap::github::api::token::create() {
 
   # we have a token now - save it to secure storage:
   strap::github::token::save "$username" "$token"
+
+  #Test if we need to do an SSO enable
+  response="$(curl --silent --show-error -i -H "Authorization: token $token" -H 'Content-Type: application/json' https://api.github.com/user/issues)"
+  headers="$(echo "$response" | sed "/^\s*$(printf '\r')*$/q" | sed '/^[[:space:]]*$/d' | tail -n +2)"
+  if echo "$headers" | grep -q 'X-GitHub-SSO:'; then
+    # Linux based system
+    if command -v gio >/dev/null; then
+      gio open https://github.com/settings/tokens
+    # Mac
+    elif command -v open >/dev/null; then
+      open https://github.com/settings/tokens
+    else
+      echo "Cannot detect command to open browser! Please open https://github.com/settings/tokens"
```

Comment:
Now that I think about it, I'm not sure we need this last else statement if we make the echo lines obvious, like this:

```bash
echo 'One or more of your SSO-enabled GitHub organizations requires the Strap-generated GitHub Auth Token to be authorized before using it.  Please visit https://github.com/settings/tokens in your web browser and authorize the token'
echo
echo 'For more information, please read https://help.github.com/en/articles/authorizing-a-personal-access-token-for-use-with-a-saml-single-sign-on-organization.'
echo
read -r -p "Press [Enter] key once you have enabled the token..."  ignore </dev/tty
```